src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs

```diff
@@ -321,63 +323,6 @@ private async Task<Bitmap> FetchResourceWithDownsampling(CancellationToken ct, i
 			return bitmap;
 		}
 
-		#region Resources
-		private static Dictionary<string, int> _drawablesLookup;
-		private static Type _drawables;
-		private int? _resourceId;
-
-		public static Type Drawables
-		{
-			get
-			{
-				return _drawables;
-			}
-			set
-			{
-				_drawables = value;
-				Initialize();
-			}
-		}
-
-		private static void Initialize()
-		{
-			_drawablesLookup = _drawables
-				.GetFields(BindingFlags.Static | BindingFlags.Public)
-				.ToDictionary(
-					p => p.Name,
-					p => (int)p.GetValue(null)
-				);
-		}
-
-		/// <summary>
-		/// Returns the Id of the bundled image.
-		/// </summary>
-		/// <param name="imageName">Name of the image</param>
-		/// <returns>Resource's id</returns>
-		public static int? FindResourceId(string imageName)
```

Comment:
Those methods and properties cannot be removed, as the methods are used by existing users of Uno. The body of the methods can redirect to the new location, though. You can mark those as editor hidden, though.

Reply:
@jeromelaban  Should be fixed now - returned both these methods and the previous `DrawableHelper`. One more question though - should the `DrawableHelper` stay directly in the `Uno` namespace or would `Uno.Android` or `Uno.Helpers` be more appropriate?

Reply:
Let's keep them in the `Uno.UI` namespace, Uno is too broad for this assembly. Using `Android` as a namespace will cause conflicts :)

Reply:
The problem is there is already the Uno.UI.DrawableHelper in the Uno.UI library (we now have `src/Uno.UWP/DrawableHelper.Android.cs` and `src/Uno.UI/DrawableHelper.Android.cs`), so this should probably be something else or it would clash

Reply:
Alternative would also be to rename the new one to something else than `DrawableHelper`

Reply:
Alternative, could also be to remove DrawableHelper from Uno.UI completely, but I guess that would be the same problem as with the ImageSource methods, as it was public and some users may be using.

Reply:
@jeromelaban Just pinging, do you have some suggestion what to do with this problem, please? I would like to close this one soon so I can use it in #1306

Reply:
@davidjohnoliver I learned Jerome is currently out of office, any chance you could help resolve this namespace problem?

Reply:
Sure thing - the main concern in this case is not introducing a binary breaking change, beyond that the namespace isn't such a big deal, since most consumers won't be using this code directly. 

My call on this:
 - leave `Uno.UI.DrawableHelper` and Uno.DrawableHelper` as you currently have, with `Uno.UI.DrawableHelper` internally redirecting to `Uno.DrawableHelper`
 - mark `Uno.DrawableHelper` and its methods as `EditorBrowsableState.Never`, to minimise confusion

I think that should do it.

Reply:
So I would keep `Uno.DrawableHelper` public, in this general namespace? I wouldn't want to clutter it though, as it is currently empty otherwise I think. Key thing is it has to be public and ideally it should be the "go to helper for Drawables", so I would think it could be better to make the existing UI.DrawableHelper editor hidden instead

Reply:
>  I would think it could be better to make the existing UI.DrawableHelper editor hidden instead (to cover the unlikely scenario when some library references Uno dll but not Uno.UI dll)

Fair enough, let's hide the `Uno.UI` one then.

For the class in the Uno dll, let's call it `Uno.Helpers.DrawableHelper`.

Reply:
Alright, sounds good :-)

Reply:
@davidjohnoliver pushed the requested changes